tests/baselines/reference/project/baseline3/amd/nestedModule.js

```diff
@@ -1,11 +1,11 @@
 define(["require", "exports"], function (require, exports) {
     "use strict";
-    var outer;
     (function (outer) {
         var inner;
         (function (inner) {
             var local = 1;
             inner.a = local;
         })(inner = outer.inner || (outer.inner = {}));
-    })(outer = exports.outer || (exports.outer = {}));
+    })(exports.outer || (exports.outer = {}));

```

Comment:
this change is the emit is unnecessary (given that emit for non-exported modules still matches the old emitter) , I'll keep it from merging for now